azure-devops/azext_devops/dev/pipelines/commands.py

```diff
@@ -44,6 +44,12 @@ def load_build_commands(self, _):
         g.command('queue', 'build_queue', table_transformer=transform_build_table_output)
         g.command('show', 'build_show', table_transformer=transform_build_table_output)
 
+    with self.command_group('pipelines build tags', command_type=buildOps) as g:
+        # basic build tag commands
+        g.command('list', 'get_build_tags', table_transformer=transform_tasks_table_output)
```

Comment:
Will the task table output work? I think you will need to define the table format for this..